src/main/java/cam72cam/immersiverailroading/entity/LocomotiveSteam.java

```diff
@@ -664,8 +664,8 @@ public void onUpdate() {
 		} else {
 			if (boilerPressure > 0) {
 				// Reduce pressure by needed temperature
-				boilerPressure = Math.max(0, boilerPressure - (100 - boilerTemperature));
-				boilerTemperature = 100;
+				//boilerPressure = Math.max(0, boilerPressure - (100 - boilerTemperature));
```

Comment:
Why comment this out?  This keeps some of the core (admittedly strange) mechanics working

Reply:
It's the only way that I found to get the Steam Loader working. When it's not commented out the Pressure gets always set to 0 if the Steam Loco isn't fired up. I didn't find any other fix for that.

Reply:
![2018-08-31_18-37-12](https://user-images.githubusercontent.com/22636066/44924891-00a99180-ad4d-11e8-862e-59d9bd8f1b8a.gif)
That happens with those lines.

Reply:
Ah, I see.  Maybe it's time to rewrite that code to be more realistic.

Reply:
That code needs to condense steam which should increase the boiler heat till it hits 100